runtime/drivers/slack/slack.go

```diff
@@ -72,7 +72,22 @@ var _ drivers.Handle = &handle{}
 
 // Ping implements drivers.Handle.
 func (h *handle) Ping(ctx context.Context) error {
-	return drivers.ErrNotImplemented
+	if h.config.BotToken == "" {
+		return fmt.Errorf("bot token not configured")
+	}
```

Comment:
I think it's optional? Not required for Slack webhooks I believe. So it should just `return nil` if no token is set.